### DISCUSSION TITLE
How best to use lodash in a Nuxt project?

### DISCUSSION THREAD

**Discussion by justindasilva** [nuxt/nuxt#18297] [Questions]
It seems that while I try to use lodash through Nuxt Bridge, I'm getting some errors.

```
import { first } from 'lodash';
  ^^^^^
  SyntaxError: Named export 'first' not found. The requested module 'lodash' is a CommonJS module, which may not support all module.exports as named exports.
  CommonJS modules can always be imported via the default export, for example using:
```

The workaround seems to be to do this:
```
import pkg from 'lodash';
const { first } = pkg;
```

Is there some way I can continue using the method I was using above?
I actually tried the `lodash-es` module, but ran into other errors with it.

For now, the workaround is working, but I miss that simpler syntax.

**Comment by wobsoriano** [comment]
Have you tried transpiling it?

```ts
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
    build: {
        transpile: ['lodash']
    }
})
```

**Reply by justindasilva** [reply]
That did not work for me.
